modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -36,28 +36,18 @@ const fadeOutAnimation = keyframes`
 
 export const menuFadeDuration = 200;
 
-const menuBorderCSS = (error?: ErrorType, theme?: EmotionCanvasTheme): CSSObject => {
-  let borderColor = inputColors.focusBorder;
+const menuBorderCSS = (theme: EmotionCanvasTheme, error?: ErrorType): CSSObject => {
+  let borderColor = theme.canvas.palette.common.focusOutline;
   let dividerBorderColor = borderColor;
   let dividerBorderWidth = 1;
+
   if (error === ErrorType.Error) {
-    if (theme) {
-      borderColor = theme.canvas.palette.error.main;
-      dividerBorderColor = theme.canvas.palette.error.main;
-    } else {
-      borderColor = inputColors.error.border;
-      dividerBorderColor = inputColors.error.border;
-    }
+    borderColor = theme.canvas.palette.error.main;
+    dividerBorderColor = borderColor;
   } else if (error === ErrorType.Alert) {
-    if (theme) {
-      borderColor = theme.canvas.palette.alert.main;
-      dividerBorderColor = theme.canvas.palette.alert.main;
-      dividerBorderWidth = 2;
-    } else {
-      borderColor = colors.cantaloupe600;
-      dividerBorderColor = inputColors.warning.border;
-      dividerBorderWidth = 2;
-    }
```

Comment:
if no theme is provided, there's no default color which is why i initially added the if!

Reply:
`theme` should always be provided now that it's a required argument to the helper function.